src/docs/getting-started/quick-start.md

```diff
@@ -1,72 +1,92 @@
 # Quick start
 
+This guide covers how you can quickly get started using Sealer.
+
+## PREREQUISITES
```

Comment:
Prerequisite?

Reply:
use plurality here

Reply:
i mean ,do we need lower case for the titles word?